sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -320,6 +320,8 @@ object FunctionRegistry {
     expression[Stack]("stack"),
     expression[CaseWhen]("when"),
 
+    expression[TryAdd]("try_add"),
+    expression[TryDivide]("try_divide"),
```

Comment:
If these functions exist in the `math` function group, how about moving them after L376?

Reply:
Done